core/lib/genapi.pl

```diff
@@ -212,6 +212,11 @@ sub copy_file
     push(@headers, "$core/annotations.h");
 }
 
+# AArch64's opcode.h is auto-generated.
+if (defined($defines{"AARCH64"})) {
+    push(@headers, "$dir/../opcode.h");
```

Comment:
So this assumes the output dir is one deep in the build dir.  Worth a comment about the assumption.